### ISSUE TITLE
Refine bug reporting guidance documentation

### ISSUE THREAD

**Issue by jmcook1186** [Green-Software-Foundation/if#590]
**What**
Update the contribution guidelines in our main repository and docs website to help users report bugs effectively.

**Why**
As a core dev I want our users to submit bug reports in a standard format so we can triage it effectively and apply our decision making process consistently.

As a user, I want a clear explanation of what's expected when I report a bug so I can be confident it will be handled effectively.

We can make our bug response much more efficient and effective by clearly communicating to users what information we need to reproduce, diagnose and fix bugs. If users conform to our standard, we will be able to handle them in our triage calls without requiring lots of back and forth async discussion with the user.  We are updating our bug report template in a separate ticket and this issue adds explicit guidance on how to fill out the template to our contribution documentation in the IF repository and our docs website to make sure users can provide us the information we need to address their issues efficiently and effectively.

NOTION LINKS:

**Statement of Work**

- [ ] Write up guidance documentation for filling out our new bug report template on `if/contributing.md`
~- [ ] Write up guidance documentation for filling out our new bug report template on the contributing page at if.greensoftware.foundation~(one source of truth)
- [ ] Change if contributions documentation to point to `if/contributing.md`

**Prerequisites**

#586 must be completed first

**Acceptance criteria**

- [ ] Guidance includes the relevant discussions in the epic discussion doc here: https://www.notion.so/grnsft/IF-Epic-Build-trust-in-the-impact-framework-ecosystem-3800ec4e76314709ba4362523e121485?pvs=4#f4c73b8ac8d54fc280d1140c3f2096a3
- [ ] Guidance has been reviewed by the team.

**Deadline**
tbc

*Reviewer**

**Comment by jawache** [comment]
@jmcook1186 is there a strong reason to keep this separate from #586? The guidance and template would be done hand in hand I'd have the same conversations on both tickets. To me it makes sense to merge into one, it's all going to be done by the same person anyway and one is immediately after the other.

**Comment by jmcook1186** [comment]
Sure, they can be combined. I'll close this and add there.

**Comment by zanete** [comment]
Closing as per above conversation
